kubernetes-elasticsearch-cluster/deploy-es.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I get this error with 1000 Genomes:
```
~/data-explorer-indexers (bc/query-performance): kubernetes-elasticsearch-cluster/deploy-es.sh 1000_genomes_melchang_project
Updated property [core/project].
ERROR: (gcloud.container.clusters.get-credentials) argument --zone/-z: expected one argument
Usage: gcloud container clusters get-credentials NAME [optional flags]
  optional flags may be  --help | --internal-ip | --region | --zone

For detailed information on this command and its flags, run:
  gcloud container clusters get-credentials --help
```
You can run this to unset default zone, so you see these problems as well.
```
gcloud config unset compute/zone
gcloud config unset compute/region
```

Reply:
I believe this is because I left in `elasticsearch-cluster-testing` and you don't have a cluster named that (which is where we get the zone). The `get-credentials` command does pass in the the zone correctly.

Thanks, that's very useful so I don't run into these issues anymore.